docs/statcast_fielding.md

```diff
@@ -0,0 +1,123 @@
+# Statcast Fielding Outs Above Average
+`statcast_outs_above_average(year: int, pos: str, min_att: Union[int, str] = "q")`
+
+This function retrieves outs above average (OAA) for the given year, position, and attempts. OAA is a Statcast metric based on the "cumulative effect of all individual plays a fielder has been credited or debited with, making it a range-based metric of fielding skill that accounts for the number of plays made and the difficulty of them".
+
+## Arguments
+`year:` The year for which you wish to retrieve batted ball against data. Format: YYYY.
+`pos:` The position you are interested in. Valid positions include "all", "IF", "OF", and position names or abbreviations. Pitchers and catchers are not included.
+`min_att:` The minimum number of fielding attempts for the player to be included in the result. The default is players, which is 1 fielding attempt per game played for 2B, SS, 3B, and OF and 1 fielding attempt per every other game played for 1B.
+
+## Examples of Valid Queries
+```python
+from pybaseball import statcast_outs_above_average
+
+# All fielders with at least 50 fielding attempts in 2019
+data = statcast_outs_above_average(2019, "all", 50)
+
+# Center fielders who qualified in 2019
+data = statcast_outs_above_average(2019,  pos = "CF")
+
+# Infielders with at least 100 fielding attempts in 2019
+data = statcast_outs_above_average(2019, pos = "IF", min_att = 100)
+```
+
+# Statcast Fielding Outfield Directional OAA
+`statcast_outfield_directional_oaa(year: int, min_opp: Union[int, str] = "q")`
+
+This function retrieves outfielders' directional OAA data for the given year and number of opportunities. The directions are Back Left, Back, Back Right, In Left, In, and In Right.
+
+## Arguments
+`year:` The year for which you wish to retrieve batted ball against data. Format: YYYY.
+`min_opp:` The minimum number of opportunities for the player to be included in the result. The default is players with at least 1 fielding attempt per game.
+
+## Examples of Valid Queries
+```python
+from pybaseball import statcast_outfield_directional_oaa
+
+# All qualified outfielders from 2019
+data = statcast_outfield_directional_oaa(2019)
+
+# All outfielders with at least 200 attempts in 2019
+data = statcast_outfield_directional_oaa(2019, 200)
+```
+
+# Statcast Fielding Outfield Catch Probability
+`statcast_outfield_catch_proba(year: int, min_opp: Union[int, str] = "q")`
```

Comment:
Wouldn't 'prob' be a more normal condensation (and hence `statcast_outfield_catch_prob`)?

Reply:
Yeah it could be. I'm used to scikit-learn's usage where they have `predict_proba()` on things like the [Random Forest Classifier](https://scikit-learn.org/stable/modules/generated/sklearn.ensemble.RandomForestClassifier.html#sklearn.ensemble.RandomForestClassifier.predict_proba)